### PR TITLE
Improved TOC styling (and minor improvements)

### PR DESCRIPTION
**Pull request by j9t**
As per DX-76, the solution clamps the image size (which gets the TOC to show consistently), and then hides the TOC in small viewports.

### CODE REVIEW

File: css/default.css [hunk 1]
```diff
@@ -556,6 +556,10 @@ table tbody td img {
         max-width: 100%;
         margin: inherit;
     }
+
+    #jumpto {
```
Comment (j9t) [inline css/default.css:560]: @itay1313 FYI (just to point to the more interesting pieces)

File: css/inner-page.css [hunk 2]
```diff
@@ -38,13 +37,11 @@ a:focus-visible {
 img {
     vertical-align: middle;
     margin: 24px 0;
-    max-width: 800px;
+    max-width: clamp(100px, 100%, 800px);
```
Comment (j9t) [inline css/inner-page.css:40]: @itay1313 FYI

File: css/default.css (hunk 1 above)
Comment (itay1313) [reply css/default.css:560]: Good one

Comment (itay1313) [reply css/default.css:560]: we can also delete this jumpto.css("display", "flex") add display flex in CSS and not use !important but it was like that anyway in 767 so it's cool

Comment (j9t) [reply css/default.css:560]: Would be awesome if you like to give this another go afterwards, and clean up! Didn’t want to meddle with this so took this route.

**Review by itay1313 — APPROVED** [review]

File: css/inner-page.css (hunk 2 above)
Comment (itay1313) [reply css/inner-page.css:40]: 👌

**Pull request merged**